tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -71,10 +71,10 @@
       <PackageFile Include="$(MSBuildThisFileDirectory)Package.Default.targets" Condition="Exists('$(MSBuildThisFileDirectory)Package.Default.targets')">
         <PackagePath>build\$(PackageId).Default.targets</PackagePath>
       </PackageFile>
-      <PackageFile Include="$(MSBuildProjectDirectory)\$(PackageId).Custom.targets" Condition="Exists('$(MSBuildProjectDirectory)\$(PackageId).Custom.targets')">
```

Comment:
It has to be named the PackageId for the targets & props to automatically be imported when the package is included. See [Including MSBuild props and targets in a package](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#including-msbuild-props-and-targets-in-a-package).